src/Backend/Modules/Pages/Domain/Page/Form/PageNavigationType.php

```diff
@@ -31,8 +37,22 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'label_attr' => [
                     'class' => 'sr-only',
                 ],
+                'constraints' => [
+                    new NotBlank(),
+                ]
             ]
         );
+
+
+        $builder->addEventListener(FormEvents::PRE_SET_DATA, function (FormEvent $formEvent) {
+            $data = $formEvent->getData();
+            if (array_key_exists('titleOverwrite', $data)) {
+                return;
+            }
+
+            $data['title'] = $formEvent->getForm()->getParent();
+            $formEvent->getForm()->getParent()->getData();
```

Comment:
What does this do?

Reply:
fixed 23896fdbab8db62d9abfe116ee2d8af10b8f5dad